test/index.ts

```diff
@@ -129,17 +130,33 @@ describe('Login Mutation', () => {
       password: correctInputUser.password,
     };
 
-    const mutation = await loginMutation(url, loginCredentials);
+    const mutationWithRememberOn = await loginMutation(url, { ...loginCredentials, rememberMe: true });
 
-    const resultData = mutation.data.data.login;
+    const rememberLoginData = mutationWithRememberOn.data.data.login;
 
-    expect(resultData.token).to.not.be.empty;
+    expect(rememberLoginData.token).to.not.be.empty;
+
+    const rememberToken = verify(rememberLoginData.token, 'supersecret') as JwtPayload;
+
+    const mutationWithRememberOff = await loginMutation(url, { ...loginCredentials, rememberMe: false });
+
+    const normalLoginData = mutationWithRememberOff.data.data.login;
+
+    expect(normalLoginData.token).to.not.be.empty;
+
+    const normalToken = verify(normalLoginData.token, 'supersecret') as JwtPayload;
+
+    const expirationWithoutRemember = normalToken.exp - normalToken.iat;
+    const expirationWithRememeber = rememberToken.exp - rememberToken.iat;
+
+    expect(expirationWithRememeber > expirationWithoutRemember).to.be.true;
```

Comment:
Não sei se foge do escopo do onboard, mas acho que da pra começar a aplicar um AAA nesses its. (Se fugir, fica a dica pra depois)

Na arquitetura dos its, da pra organizar por hierarquia (separados por um espaço) 
1. setup inicial do teste (declaração de variáveis, stub de métodos, etc)
2. execução do objetivo do teste (chamar a função sendo testada, requisição, etc)
3. verifica se a condição do it foi satisfeita (chama todos os expect)

Entao no fim das contas, vc terá três blocos de código por it, e se vc precisar de mais do que isso, às vezes vale a pena criar outro it.